README.md

```diff
@@ -104,14 +97,14 @@ There is one project in this sample. To register it, you can:
 
 </details>
 
-### Choose the Azure AD tenant where you want to create your applications
+### Step 2 (Alternative Manual setup): Choose the Azure AD tenant where you want to create your applications
```

Comment:
@aremo-ms do we need this heading? the section above (previous step 2) reads as:

> There is one project in this sample. To register it, you can:
> 
> follow the steps below for manually register your apps
> or use PowerShell scripts that:
> automatically creates the Azure AD applications and related objects (passwords, permissions, dependencies) for you.
> modify the projects' configuration files.
> Expand this section if you want to use this automation: